backend/router/routes.js

```diff
@@ -36,11 +38,19 @@ const rides = [
 router.get('/', (req, res) => {
   res.status(200).json({ message: 'Hello world' });
 });
+router.post('/', (req, res) => {
+  res.send('Got a POST request');
+});
 
 router.get('/rides', (req, res) => {
   res.status(200).json({ rides });
 });
 
+router.post('/rides', (req, res) => {
+  rides.push(req.body.newRide);
```

Comment:
I am assuming you have to send a post request as in this format right?
``` js
{
 newRide { /*ride information here*/}
}
```

Reply:
Yes, that is correct